### PR TITLE
Adding a section on estimators

### PR DESCRIPTION
**Pull request by Shotgunosine**

### CODE REVIEW

File: docs/estimators.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.. include:: links.rst
+
+Estimators
+==========
+Fitlins supports two estimators for first level models,
+Nilearn's :py:class:`nilearn.glm.first_level.FirstLevelModel`
+and AFNI's :ref:`3dREMLfit<afni:ahelp_3dremlfit>`.
+In both cases, the design matrix is generated by Nilearn.
```
Comment (adelavega) [inline docs/estimators.rst]: Well, I would be more specific and say it's generated by pybids + nilearn. pybids really does most of the work in design matrix generation actually.

File: docs/estimators.rst [hunk 2]
```diff
@@ -0,0 +1,19 @@
+.. include:: links.rst
+
+Estimators
+==========
+Fitlins supports two estimators for first level models,
```
Comment (adelavega) [inline docs/estimators.rst]: We should also explicitly mention that Nilearn is the only option for higher-level models

**Review by adelavega — CHANGES_REQUESTED** [review]
Good addition! Some comments

File: docs/conf.py [hunk 3]
```diff
@@ -57,6 +57,11 @@
 
 source_suffix = ['.rst', '.md']
 
+intersphinx_mapping = {
+    'nilearn': ('https://nilearn.github.io/', None),
+    'afni': ('https://afni.nimh.nih.gov/pub/dist/doc/htmldoc/', None)
```
Comment (effigies) [inline docs/conf.py:62]: Huh. That's really cool.

File: docs/estimators.rst (hunk 2 above)
Comment (effigies) [reply docs/estimators.rst]: Done in my changes.

**Comment by effigies** [comment]
@Shotgunosine @adelavega What do you think? I'm not worried about the failing test. Seems one-off and other branches are passing.

Rendered version at https://fitlins--375.org.readthedocs.build/en/375/estimators.html

**Comment by Shotgunosine** [comment]
@effigies, looks good to me!

**Comment by effigies** [comment]
Thanks! And thanks for the first pass on this doc.

**Pull request merged**